lib/simulator.js

```diff
@@ -92,6 +92,15 @@ const devicePairCompatiblity = exports.devicePairCompatiblity = {
 		'9.x': {        // iOS 9.0
 			'2.x': true // watchOS 2.0
 		}
+	},
+	'8.x': {
+		'>=9.0 <=9.3': {
+			'2.x': true
+		},
```

Comment:
This is not correct. You CAN pair a WatchOS 3 simulator with a iOS 9 simulator. Please add `'3.x': true`.